docs/buildah-run.md

```diff
@@ -58,5 +119,7 @@ buildah run --tty containerID /bin/bash
 
 buildah run --tty=false containerID ls /
 
+buildah run --volume /path/on/host:/path/in/container:ro,z containerID sh
+
 ## SEE ALSO
 buildah(1)
```

Comment:
If we're going to talk about Podman above, we ought to include a manpage link here.  podman(1)

Reply:
Should be buildah, fixed.